### PR TITLE
specify further the pipeline following #345

### PR DESCRIPTION
**Pull request by CPernet**
#345  discuss splitting a pipeline into multiple derivatives with/without nesting

### CODE REVIEW

File: src/05-derivatives/01-introduction.md [hunk 1]
```diff
@@ -12,22 +12,34 @@ Derivatives can be stored/distributed in two ways:
...
     ```
+    Examples of a split pipeline outputs:
+    ```Plain
+    <dataset>/derivatives/spm_preproc/sub-0001
+    <dataset>/derivatives/spm_stats/sub-0001
+    ```
+    or using a nested structure
+    ```Plain
+    <dataset>/derivatives/spm_preproc/sub-0001
+    <dataset>/derivatives/spm_preproc/sub-0001/spm_stats/sub-0001
```
Comment (effigies) [inline src/05-derivatives/01-introduction.md]: I would expect a nested structure to look like:

```suggestion
    <dataset>/derivatives/spm_preproc/derivatives/spm_stats/sub-0001
```

File: src/05-derivatives/01-introduction.md [hunk 2]
```diff
@@ -12,22 +12,34 @@ Derivatives can be stored/distributed in two ways:
...
 
-    For example:
+    Examples of single pipeline outputs:
 
     ```Plain
     <dataset>/derivatives/fmriprep-v1.4.1/sub-0001
     <dataset>/derivatives/spm/sub-0001
     <dataset>/derivatives/vbm/sub-0001
     ```
+    Examples of a split pipeline outputs:
```
Comment (effigies) [inline src/05-derivatives/01-introduction.md]:
```suggestion

    Example of a pipeline with split derivative directories:

```

File: src/05-derivatives/01-introduction.md [hunk 3]
```diff
@@ -12,22 +12,34 @@ Derivatives can be stored/distributed in two ways:
...
+    ```Plain
+    <dataset>/derivatives/spm_preproc/sub-0001
+    <dataset>/derivatives/spm_stats/sub-0001
+    ```
+    or using a nested structure
```
Comment (effigies) [inline src/05-derivatives/01-introduction.md]:
```suggestion

    Example of a pipeline with nested derivative directories:

```

File: src/05-derivatives/01-introduction.md [hunk 4]
```diff
@@ -12,22 +12,34 @@ Derivatives can be stored/distributed in two ways:
...
+    however, also be stored under different subfolders. There are few restrictions on
+    the directory names; it is RECOMMENDED to use the format `<pipeline>-<variant>` in 
+    cases where it is anticipated that the same pipeline will output more than one variant (e.g.,
     `AFNI-blurring`, `AFNI-noblurring`, etc.). For the sake of consistency, the
     subfolder name SHOULD be the `PipelineDescription.Name` field in
     `data_description.json`, optionally followed by a hyphen and a suffix (see
     below).
 
-    For example:
+    Examples of single pipeline outputs:
```
Comment (effigies) [inline src/05-derivatives/01-introduction.md]:
```suggestion
    Example of derivatives with one directory per pipeline:
```

File: src/05-derivatives/01-introduction.md (hunk 1 above)
Comment (CPernet) [reply src/05-derivatives/01-introduction.md]: oh my, i messed up my own suggestion ... thx for picking that up

**Comment by effigies** [comment]
Thanks @CPernet.

**Pull request merged**